src/components/forms/formr-part-b/viewSections/ViewSection2.cy-test.tsx

```diff
@@ -53,12 +49,12 @@ describe("View", () => {
   });
 
   it("should render the correct form data", () => {
-    testData(formDataToDisplay);
+    cy.testData(formDataToDisplay);
```

Comment:
Hi @DorisWongNHS , I'm getting an `Property 'testData' does not exist on type 'cy & EventEmitter'` error in the ViewSection cy-test files but adding the Cypress reference types and path (see below) at the top of just the `ViewSection1.cy-test.tsx` fixes it for all test files. Are you getting the same error? 
`/// <reference types="cypress" />
/// <reference path="../../../../../cypress/support/index.d.ts" />`